sec_certs/sample/common_criteria.py

```diff
@@ -27,6 +32,11 @@
 }
 
 
+class DependencyCVEType(Enum):
+    DIRECT = "direct"
+    INDIRECT = "indirect"
```

Comment:
*Nitpicking*: Shouldn't this be just `DepdendencyType`? How is it related to CVE?